### PR TITLE
don't be too strict :sweat_smile:

### PR DESCRIPTION
**Pull request by ViktorTsupa**
- [DEMO LINK](https://ViktorTsupa.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ViktorTsupa.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,54 @@
...
+            <a class="nav__is-active" href="#Samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__is-active" href="#Smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+             class="nav__is-active"
+              href="#Laptops & Computers"
+               data-qa="hover">Laptops & Computers</a>
```
Comment (etojeDenys) [inline src/index.html:40]: Pasted Graphic

**Review by etojeDenys — CHANGES_REQUESTED** [review]
your links don't work, try to fix it

**Comment by ViktorTsupa** [comment]
> your links don't work, try to fix it

done

File: src/index.html [hunk 2]
```diff
@@ -5,10 +5,54 @@
...
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo-link" href="#logo">
+        <img
+         class="logo"
+         src="images/logo.png"
+          alt="moyo logo"
+          >
```
Comment (OleziO) [inline src/index.html:23]:
```suggestion
        <img
          class="logo"
          src="images/logo.png"
          alt="moyo logo"
        >
```

File: src/index.html (hunk 1 above)
Comment (OleziO) [inline src/index.html:40]:
```suggestion
            <a
              class="nav__is-active"
              href="#Laptops & Computers"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+  font-weight: 500;
+  font-style: normal;
+  color: #000;
+  text-transform: uppercase;
+}
+
+.nav__is-active:hover::after {
+  content: '';
+  position: absolute;
+  border-bottom: 4px solid #00ACDC;
```
Comment (OleziO) [inline src/style.css:40]: Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+
+.nav__is-active:hover {
+  color: #00ACDC;
+}
+
+
+
+
+
+
```
Comment (OleziO) [inline src/style.css:55]:
```suggestion

```

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (anastasiiavorobiova) [reply src/index.html:23]: Not fixed

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider fixing the previous comments and centering your header on the page 
Screenshot 2024-01-14 at 17 05 17

**Comment by ViktorTsupa** [comment]
it's wrong PR from my branch master(old one) I created a correct one with the develop branch 2 days ago, now I will find it and post a link

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Please, use an instruction to update code in this PR and reopen it https://mate-academy.github.io/layout_task-guideline/#codegit-pushcode-to-send-your-code-to-github

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls fix prev comments

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Previous comments were not fixed, please check direct messages on the platform :)

File: src/index.html (hunk 1 above)
Comment (ViktorTsupa) [reply src/index.html]: Screenshot_1

Comment (ViktorTsupa) [reply src/index.html]: Screenshot_1

**Comment by ViktorTsupa** [comment]
Screenshot_5

File: src/style.css (hunk 3 above)
Comment (ViktorTsupa) [reply src/style.css]: Screenshot_6
Screenshot_7

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**